utils/textgeneration.go

```diff
@@ -42,7 +42,7 @@ ___
 	` **[2] - **[More Markdown Tests]**
 > i am a blockquote!
 
-> i am a 2nd multiline 
+> i am a 2nd multiline
```

Comment:
Please revert these. These are multi-line markdown strings, and the trailing space is for a reason.

Reply:
fixed

Reply:
Hmm .. I still don't see them :thinking:

Reply:
sorry, my mistake